aiosmtpd/smtp.py

```diff
@@ -78,13 +97,38 @@ def max_command_size_limit(self):
             return self.command_size_limit
 
     def connection_made(self, transport):
-        super().connection_made(transport)
-        self.peer = transport.get_extra_info('peername')
-        self.transport = transport
-        log.info('Peer: %s', repr(self.peer))
-        # Process the client's requests.
-        self.connection_closed = False
-        self._handler_coroutine = self.loop.create_task(self._handle_client())
+        if(self.transport is not None
+                and isinstance(transport, sslproto._SSLProtocolTransport)):
+            # It is STARTTLS connection over normal connection
+            self._reader._transport = transport
+            self._writer._transport = transport
+            self.transport = transport
+            # Reset state due to rfc3207 part 4.2
+            self._set_rset_state()
+            self.seen_greeting = False
+            # Do SSL certificate checking as rfc3207 part 4.1 says
+            # Why _extra is protected attribute?
+            extra = self._tls_protocol._extra
+            if hasattr(self.event_handler, 'handle_tls_handshake'):
+                self._tls_handshake_failed = \
```

Comment:
Is there a way to refactor this (possibly using a local variable) to avoid the backslash?

Also, I often find double negatives to be difficult to reason about.  For example `self._tls_handshake_failed=False` means the the handshake succeeded right?  So why not name it `_tls_handshake_succeeded` and a True value is easier to parse mentally?